### PR TITLE
Added Gridex dex to dex.trades

### PR DESCRIPTION
**Pull request by ARDev097**
Added trades data for `gridex protocol` on `arbitrum`.

### CODE REVIEW

**Comment by Hosuke** [comment]
Hi @ARDev097,
you can run `git merge main` to update this PR.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @ARDev097 

[Sample gridex_arbitrum_base_trades](https://dune.com/queries/3391526)

**Review by jeff-dude — APPROVED** [review]

**Comment by ARDev097** [comment]
Hello @couralex6,
Please go ahead and make the necessary changes to resolve the merge conflict.

File: models/_sector/dex/trades/arbitrum/_sources.yml
Comment (Hosuke) [inline models/_sector/dex/trades/arbitrum/_sources.yml:1]: Seems this PR blocked the maintainers' edits.

Hi @ARDev097,
please move the changes in this file to `sources/_sector/dex/trades/arbitrum/_sources.yml`.
This file may not be existed in main branch now.

**Pull request closed without merging**